### ISSUE TITLE
Possible to use this on an element?

### ISSUE THREAD

**Issue by johanneslamers** [koggdal/scroll-converter#7]
Is it possible to use scroll-converter on an element?

**Comment by koggdal** [comment]
No not currently, but you can make a pull request for it if you'd like! :)

**Comment by xf75013** [comment]
yes it is a very good idea to be able to restrict the triggering of scrolling on an element of the page… please

**Comment by joebentaylor** [comment]
This is exactly what i need, as on mac when you scroll with the trackpad, you get a bounce effect on screen, having this on an element thats inside a fixed element would stop this problem for me on my project. Any update on this?

**Comment by koggdal** [comment]
I don’t actively work on this project anymore, but I’m accepting pull requests if anyone wants to implement it! 👍

@joebentaylor You may also want to look at `overscroll-behavior`: https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior
